### PR TITLE
FBXLoader add support for TGA and DDS

### PR DESCRIPTION
**Pull request by looeee**
Add support for TGA and DDS to the loader.

See #13203

### CODE REVIEW

**Comment by looeee** [comment]
Do DDSTextures need any special settings? E.g. for `minFilter`, `maxFilter` ?

File: examples/js/loaders/FBXLoader.js [hunk 1]
```diff
@@ -344,7 +382,22 @@
 
 		}
 
-		var texture = loader.load( fileName );
+		var texture;
+
+		if ( fileName.indexOf( 'image/dds' ) !== - 1 ) {
+
+			texture = THREE.Loader.Handlers.get( '.dds' ).load( fileName );
+			console.log( 'dds', texture );
```
Comment (mrdoob) [inline examples/js/loaders/FBXLoader.js]: Do you want to keep this?

Comment (looeee) [reply examples/js/loaders/FBXLoader.js]: Oops, no.

**Comment by looeee** [comment]
This PR currently doesn't work since it seems dds textures can't be treated the same way as other textures types. 

I don't have any particular need to add support though, so I'll close this for now. 

If anybody else wants to have a go, this PR will make a good starting point.

**Pull request closed without merging**